OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+SizeSemanticTokens.swift

```diff
@@ -18,69 +18,69 @@ import OUDSTokensSemantic
 // swiftlint:disable identifier_name
```

Comment:
This comment has been removed in tokenator for this file. Will not be there in future update.